examples/callback-results/src/lib.rs

```diff
@@ -85,9 +85,9 @@ mod tests {
     async fn workspaces_test() -> anyhow::Result<()> {
         let wasm = fs::read("res/callback_results.wasm").await?;
 
-        let worker = workspaces::sandbox();
+        let worker = workspaces::sandbox().await?;
 
-        let contract = worker.dev_deploy(wasm).await?;
+        let contract = worker.dev_deploy(&wasm).await?;
```

Comment:
any reason not to use `include_bytes!` here?

Reply:
tried to keep this change unopinionated, can follow with that change in another